src/System.Data.SqlClient/tests/FunctionalTests/CloneTests.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Data.SqlClient.Tests
+{
+    public class CloneTests
+    {
+        [Fact]
+        public void CloneSqlConnection()
+        {
+            SqlConnectionStringBuilder builder = new SqlConnectionStringBuilder();
+            builder.DataSource = "mybadServer";
+            builder.ConnectTimeout = 1;
+            builder.InitialCatalog = "northwinddb";
+            SqlConnection connection = new SqlConnection(builder.ConnectionString);
+
+            SqlConnection clonedConnection = (connection as ICloneable).Clone() as SqlConnection;
```

Comment:
The Clone is implemented on ICloneable and it will not be available via `SqlConnection` . The SqlConnection has to be casted to ICloneable which will provide an object and that needs to be casted back to SqlConnection